src/libs/elektra/keytest.c

```diff
@@ -151,38 +154,43 @@ int keyIsUser (const Key * key)
 }
 
 /**
- * Check if the key check is below the key key or not.
+ * Check if the Key @p check is below the Key @p key or not.
  *
  * Example:
  @verbatim
  key user:/sw/app
  check user:/sw/app/key
  @endverbatim
  *
- * returns true because check is below key
+ * returns true because @p check is below @p key
  *
  * Example:
  @verbatim
  key user:/sw/app
  check user:/sw/app/folder/key
  @endverbatim
  *
- * returns also true because check is indirect below key
+ * returns also true because @p check is indirectly below @p key
  *
- * Obviously, there is no key above a namespace (e.g. user, system, /):
+ * Obviously, there is no Key above a namespace (e.g. user, system, /):
  *
  @verbatim
  key *
  check user
  @endverbatim
  *
- * @param key the key object to work with
- * @param check the key to find the relative position of
- * @retval 1 if check is below key
+ * @param key the Key object to check against
+ * @param check the Key object for which it should be checked whether it is
+ * below @p key
```

Comment:
Does `@param` work when a part of the sentence is on a new line?

EDIT: according to docs yes, until a blank line, but I have never seen this :eyes: